src/Transform/ONNX/Decompose.cpp

```diff
@@ -270,7 +331,7 @@ void DecomposeONNXToONNXPass::runOnOperation() {
   }
 #endif
 
-  if (failed(applyPartialConversion(function, target, std::move(patterns))))
+  if (failed(applyFullConversion(function, target, std::move(patterns))))
```

Comment:
Why it is needed? Just for my learning.

Reply:
Accidental change. I am also learning and testing any difference. I think that we should use Op transform instead of Conversion for this pass because it is transform within a dialect. We do not need typeconverter and target. Example llvm-project/mlir/lib/Dialect/SCF/Transforms/ForToWhile.cpp